docs/theory.rst

```diff
@@ -16,79 +22,188 @@ The filter shape can be thought of in terms of the kernel of a convolution filte
 
 where :math:`f` is the function being filtered, :math:`G` is the filter kernel, and :math:`x'` is a dummy integration variable.
 This package currently has two filter shapes: ``GAUSSIAN`` and ``TAPER``.
-For the Gaussian filter the ``filter_scale`` equals :math:`\sqrt{12}\times` the standard deviation of the Gaussian.
+
+.. ipython:: python
+
+    list(gcm_filters.FilterShape)
+
+For the ``GAUSSIAN`` filter the ``filter_scale`` equals :math:`\sqrt{12}\times` the standard deviation of the Gaussian.
 \I.e. if you want to use a Gaussian filter with standard deviation L, then you should set ``filter_scale`` equal to L :math:`\times\sqrt{12}`.
 This strange-seeming choice makes the Gaussian kernel have the same effect on large scales as a boxcar filter of width ``filter_scale``.
 Thus ``filter_scale`` can be thought of as the "coarse grid scale" of the filtered field.
 
-The definition of the "Taper" filter is more complex, but the ``filter_scale`` has the same meaning: it corresponds to the width of a qualitatively-similar boxcar filter.
-The Taper filter is more scale-selective than the Gaussian filter; it does a better job of leaving scales larger than the filter scale unchanged, and removing scales smaller than the filter scale.
-The drawback is that it requires higher computational cost for the same filter scale, and can produce negative values for the filtered field even when the unfiltered field is positive.
+We can create a Gaussian filter as follows.
+
+.. ipython:: python
+
+    gaussian_filter = gcm_filters.Filter(
+        filter_scale=4,
+        dx_min=1,
+        filter_shape=gcm_filters.FilterShape.GAUSSIAN,
+        grid_type=gcm_filters.GridType.REGULAR,
+    )
+    gaussian_filter
+
+Once the filter has been constructed, the method ``plot_shape`` can be used to plot the shape of the target filter and the approximate filter.
```

Comment:
Suggestion: Add a comment saying that the distinction between the target filter and the approximate filter will be discussed below.